client/me/two-step/index.jsx

```diff
@@ -46,60 +48,20 @@ class TwoStep extends Component {
 		doingSetup: false,
 	};
 
-	componentDidMount() {
-		debug( this.constructor.displayName + ' React component is mounted.' );
-		this.props.userSettings.on( 'change', this.onUserSettingsChange );
-		this.props.userSettings.fetchSettings();
+	isInitialized() {
+		return !! this.props.userSettings;
```

Comment:
Won't this always be `true` because `getUserSettings` will return an empty object if data is not fetched yet?